A bunch of fixes and improvements

Felix14-v2 opened the pull request:
В основном исправления благозвучности формулировок, опечаток и начало работы над созданием согласованного интерфейса (сейчас многие термины в разных местах переведены кто в лес, кто по дрова). Локально выписываю термины, варианты локализации и условия употребления, включая альтернативные контекстуальные варианты. Возможно, в будущем совместно сформируем глоссарий для будущих переводчиков.
Также немного причесал стартовую страницу (text.js).

Завтра займусь комментированием изменений (где они того требуют), чтобы создать ветки обсуждения. Скорее всего, обсуждать придётся много.

Felix14-v2 committed:
A bunch of fixes and improvements

Felix14-v2 commented on localization/i18n/ru/OrcaSlicer_ru.po:
```diff
@@ -227,7 +227,7 @@ msgstr "Окрашено с использованием прутка %1%"
 # ??? Процедура переназначения материала завершена, выполнена
 msgid "Filament remapping finished."
-msgstr "Переназначение материала завершено."
+msgstr "Катушки переназначены."
```
По какой-то причине конкретно эта строка на русском отображается в повреждённом виде (заменил одно слово на кириллицу):
<img width="479" height="124" alt="image" src="https://github.com/user-attachments/assets/bcdf21fc-157b-49fe-a00a-fa44b4b216b7" />
Возможно, какие-то приколы с кодировкой, но точной причины (и исправления) я не смог найти.

UPD: Так и есть, баг на стороне слайсера.

AndylgTom replied:
пока Материал успешно переназначен или Материал успешно заменён.

Felix14-v2 replied:
```suggestion
msgstr "Материал успешно заменён."
```

Felix14-v2 committed:
Update OrcaSlicer_ru.po

Felix14-v2 commented on localization/i18n/ru/OrcaSlicer_ru.po:
```diff
@@ -52,7 +52,7 @@ msgid "Alt + Mouse wheel"
 
 msgid "Section view"
-msgstr "Отсечение вида"
+msgstr "Сечение"
```
Сжатие строки без потерь. Взято на основе локализации КОМПАС-3D (у них прекрасно проработан глоссарий, прямо "по ГОСТу").

AndylgTom replied:
Все таки так понятнее, отсечение происходит относительно вида который у тебя сейчас. Если бы ты сам задавал параметры как в компасе, солиде, то другое дело. Да и проблемы с местом нету.

Felix14-v2 replied:
> Да и проблемы с местом нету.

Зато есть проблема со смыслом. Вид по определению невозможно отсечь, так как семантика слова "вид" – это обзор, направление просмотра, а отсекается у нас модель. `Отсечение модели` было бы корректно, но зачем? Есть простой и правильный термин из глоссария Компаса.

> Если бы ты сам задавал параметры как в компасе, солиде, то другое дело

Собственно, параметр смещения секущей плоскости относительно плоскости экрана здесь и задаётся.

AndylgTom replied:
Ладно, пусть так будет. Но этот термин я не раз слышал и находил (вроде в уроках по блендеру), да сам говорил.

Felix14-v2 commented on localization/i18n/ru/OrcaSlicer_ru.po:
```diff
@@ -85,7 +85,7 @@ msgid "Erase all painting"
 
 msgid "Highlight overhang areas"
-msgstr "Выделение нависания по углу "
+msgstr "Угол нависания"
```
В подсказке уже дублируется эта фраза, можно сократить без потери смысла.
К тому же, выделением (в значении выделения зон поддержек) занимаются инструменты рисования, а тут именно что подсветка проблемных зон, подходящих по углу. Возможно, уместнее было бы связать с "подсветить по углу", но как по мне, и так уместно.

AndylgTom replied:
Хз, хз.
Я за то, чтобы желательно, чтобы само название отражало суть, независимо от всплывающей подсказки, а тут даже нависание не упоминается, и, сказав отдельно «Угол нависания» без привязки к этому окну, вообще непонятно, о чем речь тогда идёт. Единственно, что теперь впритык надпись, ибо заменено «свеса» -> «нависания» по твоей просьбе.

Felix14-v2 replied:
> сказав отдельно «Угол нависания» без привязки к этому окну, вообще непонятно, о чем речь тогда идёт

Мне кажется, более чем понятно, у нас же рядом находится опция, продолжением которой эта строчка и является:
<img width="416" height="301" alt="image" src="https://github.com/user-attachments/assets/ba8bc4ea-9bf8-4285-96b1-693b56549670" />

> независимо от всплывающей подсказки

Про всплывающую подсказку я упомянул потому, что устранение тавтологии – ещё один аргумент за это изменение.

AndylgTom replied:
Ок, логика у нас разная

Felix14-v2 commented on localization/i18n/ru/OrcaSlicer_ru.po:
```diff
@@ -98,13 +98,13 @@ msgid "Gap area"
 
 msgid "Tool type"
-msgstr "Тип инструмента"
+msgstr "Инструмент"
```
Не понимаю, почему в оригинале "тип", по смыслу там именно что выбор конкретных инструментов.

AndylgTom replied:
без разницы, противоречия нет что 'тип'... можно написать «Выбор инструмента»

Felix14-v2 commented on localization/i18n/ru/OrcaSlicer_ru.po:
```diff
@@ -98,13 +98,13 @@ msgid "Gap area"
 
 msgid "On overhangs only"
-msgstr "Только на нависаниях"
+msgstr "Ограничить область по углу"
```
Просто как субъективный вариант на рассмотрение.

AndylgTom replied:
Пусть таки и остаётся :) Норм.

Felix14-v2 replied:
Имеется ввиду вариант `Только на нависаниях`?

AndylgTom replied:
> Имеется ввиду вариант `Только на нависаниях`?

ну да я за него )

Felix14-v2 replied:
```suggestion
msgstr "Только на нависаниях"
```

Felix14-v2 commented on localization/i18n/ru/OrcaSlicer_ru.po:
```diff
@@ -144,7 +144,7 @@ msgstr "Поверхностью на стол"
 #, boost-format
 msgid "Filament count exceeds the maximum number that painting tool supports. Only the first %1% filaments will be available in painting tool."
-msgstr "Количество пластиковых нитей превышает максимальное количество, поддерживаемое инструментом рисования. Только первые %1% материала будут доступны в инструменте для рисования."
+msgstr "Количество катушек превышает возможности инструмента. Для покраски будут доступны только первые %1%."
```
Сжатие без потери смысла. Относится к предупреждению, когда добавлено больше 16 катушек для печати.

Felix14-v2 commented on localization/i18n/ru/OrcaSlicer_ru.po:
```diff
@@ -192,11 +192,11 @@ msgstr "Показать/скрыть каркас"
 # ??? Переназначить прутки, Изменить, Повторно выбрать
 msgid "Remap filaments"
-msgstr "Перевыбрать материал"
+msgstr "Поменять катушки местами"
```
Можно ещё `Перевыбрать катушки`, но такой вариант по ушам немного режет. Проблем с местом в этой строке нет.
<img width="392" height="417" alt="image" src="https://github.com/user-attachments/assets/3f1dd5cf-81ed-422e-9b30-c3b7a8ac6425" />

AndylgTom replied:
не, ни катушка...

Не совсем подходит 'поменять местами' по смыслу, скорее типа.... Изменить\Заменить\Сменить один цвет\материал на другой

Felix14-v2 replied:
Согласен, перепроверил работу инструмента, автоматической замены не происходит. Так лучше?
```suggestion
msgstr "Поменять материал"
```

Felix14-v2 commented on localization/i18n/ru/OrcaSlicer_ru.po:
```diff
@@ -192,11 +192,11 @@ msgstr "Показать/скрыть каркас"
 # ??? Перевыбрать
 msgid "Remap"
-msgstr "Принять"
+msgstr "Заменить"
```
Как будто такой вариант уместнее, субъективно

AndylgTom replied:
ну имеется ввиду принять\подтвердить замену

Felix14-v2 replied:
Интересный момент: эта кнопка активна даже если изменений не было, поэтому уместнее будет обычное `Применить`, т.к. тут применяется в том числе текущее состояние, не обязательно изменения.

Felix14-v2 commented on localization/i18n/ru/OrcaSlicer_ru.po:
```diff
@@ -265,21 +264,22 @@ msgstr "дюйм"
 msgid "mm"
 msgstr "мм"
 
+# ??? В оригинале ошибка, подразумевается model
```
(Добавлял для себя комментарий, забыл удалить)
```suggestion
```

Felix14-v2 commented on localization/i18n/ru/OrcaSlicer_ru.po:
```diff
@@ -265,21 +264,22 @@ msgstr "дюйм"
+# ??? В оригинале ошибка, подразумевается model
 msgid "Part selection"
-msgstr "Выбор элемента"
+msgstr "Выбрать модель"
```
"Элемент" уже занято некоторыми настройками печати (например, `минимальный размер элемента [для печати]`)
```suggestion
msgstr "Выбрать часть"
```

Felix14-v2 commented on localization/i18n/ru/OrcaSlicer_ru.po:
```diff
@@ -265,21 +264,22 @@ msgstr "дюйм"
 
 msgid "Fixed step drag"
-msgstr "Фиксированный шаг перемещения"
+msgstr "Сместить с шагом"
```
Строго говоря, шаг там не фиксированный, – у инструмента масштабирования там шаг вообще в процентах. Думаю, правильнее будет упомянуть только шаг смещения точки.

AndylgTom replied:
Ок, пока так.
Как-то не придавал значение, что в разных инструментах используется (Перемещение и в Масштабе).
Мож, какой вариант еще придет...

*Суть... Перетаскивание маркеров гизмо с фиксированным шагом (1мм там) в одном случае для перемещения модели, в другом для изменения ее масштаба.

Felix14-v2 commented on localization/i18n/ru/OrcaSlicer_ru.po:
```diff
@@ -265,21 +264,22 @@ msgstr "дюйм"
 # Масштабирование с одной стороны
 msgid "Single sided scaling"
-msgstr "Одностороннее масштабирование"
+msgstr "Сместить без привязки к центру"
```
Оба варианта неплохие, но этот интуитивно понятен "с первого прочтения" и проще передаёт суть.

AndylgTom replied:
А мне наоборот не понятно, кого, куда сместить 🤔 ... ключевое слово здесь должно быть масштаб.
«Масштабирование с одной стороны», «Масштабирование в одном направлении» и то лучше, но длиннее

Felix14-v2 replied:
> А мне наоборот не понятно, кого, куда сместить 🤔

Ну как же – там в инструменте единственный орган ручного управления – это габаритные маркеры:

https://github.com/user-attachments/assets/69fe9a87-b1e9-42cc-b77f-49d3d2495e48

Просто по умолчанию они смещаются равномерно с привязкой к центру, а при удерживании Ctrl – без привязки. Был ещё такой вариант:
```suggestion
msgstr "Масштабирование без привязки к центру"
```
Но он длиннее. Впрочем, здесь не критично.

Felix14-v2 commented on localization/i18n/ru/OrcaSlicer_ru.po:
```diff
@@ -265,21 +264,22 @@ msgstr "дюйм"
 
 msgid "Position"
-msgstr "Позиция"
+msgstr "Координаты"
```
Просто чтобы избежать тавтологии (позиция-позиция):
<img width="534" height="113" alt="image" src="https://github.com/user-attachments/assets/3fcde482-efdd-4e22-8b96-d5932a42b8a8" />

Felix14-v2 replied:
Уже не актуально, т.к. вместо `Относительная позиция` вернули оригинальную конструкцию. Возможно, лучше и тут отменить изменения.

Felix14-v2 commented on localization/i18n/ru/OrcaSlicer_ru.po:
```diff
@@ -265,21 +264,22 @@ msgstr "дюйм"
 
 msgid "Rotate (relative)"
-msgstr "Вращение (относительное)"
+msgstr "Относительное"
```
Можно сократить, т.к. в строке выше уже используется связанное по смыслу слово:
<img width="492" height="165" alt="image" src="https://github.com/user-attachments/assets/5f9445c7-45d0-4eed-b454-1ffc25f179fe" />

AndylgTom replied:
не, положение - не значит вращение, мне не понятно

Felix14-v2 replied:
Положение – это общий термин, он обозначает и позицию, и ориентацию. Но вот в этом месте всё равно пока каша, попробую переделать на что-то более логичное.

AndylgTom replied:
> Положение – это общий термин, он обозначает и позицию, и ориентацию. Но вот в этом месте всё равно пока каша, попробую переделать на что-то более логичное.

Только это ника не проясняет того, что относится именно к вращению... будь проще.

Felix14-v2 commented on localization/i18n/ru/OrcaSlicer_ru.po:
```diff
@@ -309,22 +309,22 @@ msgid "Reset Rotation"
 
 msgid "Rotate (absolute)"
-msgstr "Вращение (абсолютное)"
+msgstr "Абсолютное"
```
Аналогично строке 282

Felix14-v2 commented on localization/i18n/ru/OrcaSlicer_ru.po:
```diff
@@ -309,22 +309,22 @@ msgid "Reset Rotation"
 
 msgid "Object coordinates"
-msgstr "СК модели"
+msgstr "Относительная позиция"
```
Не вижу смысла тут сокращать, только ухудшает восприятие. Тем более, что система координат там начинается не от центра выбранной модели, а от центра сборки:
<img width="724" height="656" alt="image" src="https://github.com/user-attachments/assets/dac0ab12-0d12-42af-a400-f6add9a55208" />

Felix14-v2 commented on localization/i18n/ru/OrcaSlicer_ru.po:
```diff
@@ -309,22 +309,22 @@ msgid "Reset Rotation"
 
 msgid "World coordinates"
-msgstr "Мировая СК"
+msgstr "Положение на столе"
```
Аналогично строке 312. Однако тут используется более универсальное `положение`, т.к. строка используется и в инструменте вращения.

Felix14-v2 commented on localization/i18n/ru/OrcaSlicer_ru.po:
```diff
@@ -309,22 +309,22 @@ msgid "Reset Rotation"
 
 msgid "Reset current rotation to the value when open the rotation tool."
-msgstr "Сброс текущего вращения до значения, установленного при открытии инструмента вращения."
+msgstr "Сбросить ориентацию до предыдущих значений"
```
Значительно упростил нагромождённую смысловую конструкцию.

AndylgTom replied:
Да все нормально и чётко понятно. А так... до каких предыдущих значений то? Мне так не понятно

Felix14-v2 replied:
> до каких предыдущих значений то? Мне так не понятно

Ну, буквально до предыдущих. У нас уже были внесены изменения в положение модели, затем мы открываем инструмент и что-то ещё меняем. И у нас отображается 2 кнопки: одна сбрасывает все изменения, а вторая – только последние, восстанавливая предыдущее положение.
В принципе, можно и так:
```suggestion
msgstr "Сбросить последние изменения ориентации"
```
Даже компактнее получится.

Felix14-v2 commented on localization/i18n/ru/OrcaSlicer_ru.po:
```diff
@@ -352,7 +352,7 @@ msgid "Dowel"
 
 msgid "Snap"
-msgstr "Клипса"
+msgstr "Защёлка"
```
Оно выглядит как защёлка, работает как защёлка, переводится как защёлка...
<img width="912" height="404" alt="image" src="https://github.com/user-attachments/assets/8cd91a88-5b9f-4edb-b658-e245342dd750" />

AndylgTom replied:
Только так никто не говорит. Клипса есть клипса. Введи Защёлка в гугле и погляди что отображается в окне, врядли ты найдешь изображения в этом значении. А потом введи клипса и почувствуй разницу, их куча разновидностей <img width="800" height="240" alt="image" src="https://github.com/user-attachments/assets/5c178681-8102-4445-8da5-1decf7338003" />

Хотя, может у  меня другая логика и другие ассоциации с защелкой.

Просто, если придираться, то защелка, так сказать от слова защелкивать, т.е. ответная часть при вставке попадает с паз\утапливается типа как на картинке, детали фиксируются с помощью 'защёлкивающегося замка'.
https://formlab.ru/sites/default/files/media_image/2023-03/f19c853aa193c5d4d1ae0a1ff217574f-bf152e9b42be7e6d6b708db7e652f2589bd147d2.png
https://formlab.ru/sites/default/files/media_image/2023-03/b9e7c01badf58168bfb74462e8f0e642-72bbf5851b4f38f4dda23450ea03e691e36eba5e.gif

А тут если в разрезе посмотреть, 
<img width="1331" height="775" alt="Снимок экрана 2025-09-14 111820" src="https://github.com/user-attachments/assets/ef13ada5-f931-44b3-a3aa-ee4aabb38259" />
там просто дырка в которою вставляется распирающая клипса (или как по другому это назвать? у меня автомобильные ассоциации), без какого либо защёлкивающегося замка.

Felix14-v2 replied:
Согласен, изучил семантику лучше, изначальный вариант вполне уместен. Сейчас верну.
```suggestion
msgstr "Клипса"
```

Felix14-v2 commented on localization/i18n/ru/OrcaSlicer_ru.po:
```diff
@@ -373,7 +373,7 @@ msgid "Place on cut"
 
 msgid "Flip upside down"
-msgstr "Перевернуть вверх дном"
+msgstr "Перевернуть"
```
Думаю, смысл не теряется

AndylgTom replied:
как будто уже не используется строка

Felix14-v2 commented on localization/i18n/ru/OrcaSlicer_ru.po:
```diff
@@ -398,7 +398,7 @@ msgstr "Высота"
 #. TRN - Input label. Be short as possible
 msgid "Rotation"
-msgstr "Вращение"
+msgstr "Поворот"
```
```suggestion
msgstr "Наклон"
```
Тут речь идёт именно о изменении угла на плоскости текста, а не вращении в пространстве. За вращение текста отвечает отдельный инструмент. Думаю, в таком контексте `Наклон` подходит лучше.
<img width="1107" height="716" alt="image" src="https://github.com/user-attachments/assets/898c6d9d-aacd-41fb-bba8-6ff319542fb8" />

Felix14-v2 committed:
Update OrcaSlicer_ru.po

Felix14-v2 commented on localization/i18n/ru/OrcaSlicer_ru.po:
```diff
@@ -309,22 +309,22 @@ msgid "Reset Rotation"
 
 msgid "Reset current rotation to real zeros."
-msgstr "Сброс текущего вращения на нулевое значение."
+msgstr "Сбросить ориентацию"
```
Аналогично, можно значительно упростить без потери смысла.

AndylgTom replied:
Только вот зачем упрощать более развёрнутый оригинальный вариант

Felix14-v2 replied:
Потому что это избыточная информация. Есть глобальный сброс – он всегда один и без вариантов, если иное не указано.

Но всё же добавлю упоминание состояния, до которого производится сброс.

Felix14-v2 commented on localization/i18n/ru/OrcaSlicer_ru.po:
```diff
@@ -309,22 +309,22 @@ msgid "Reset Rotation"
 
 msgid "Part coordinates"
-msgstr "СК элемента"
+msgstr "Координаты части"
```
Судя по всему, нигде не используется. В коде встречается лишь единожды (похоже, что во внутренней логике).
<img width="1300" height="257" alt="image" src="https://github.com/user-attachments/assets/9f2e44ba-eb34-44d6-96de-d4d8debcb528" />

AndylgTom replied:
Почему, используется в Масштабе.
<img width="1322" height="502" alt="image" src="https://github.com/user-attachments/assets/296492aa-83ea-4602-92d0-f1769fb990ee" />

Я хотел сохранить термин СК, МСК и т.д. + не делать ее такой длинной. Плюс у меня тогда в Пруше не влезет надпись - Размер [МСК] , как тогда ее писать? как Размер [КЧ]  что ли? 😕
<img width="476" height="254" alt="image" src="https://github.com/user-attachments/assets/153fde7b-86b4-40a0-be7c-0dbc4656068c" />

Felix14-v2 replied:
О как, не догадался, что надо в сборку объединить. Тогда такой вариант:
<img width="881" height="659" alt="{F2D7DBD3-FD7C-4555-A151-45CC5B75E231}" src="https://github.com/user-attachments/assets/29d55e0b-7c48-4ec3-a6c5-57cccae1b362" />
```suggestion
msgstr "Относительно части"
```
В Prusa тоже должно хватить места.

Felix14-v2 commented on localization/i18n/ru/OrcaSlicer_ru.po:
```diff
@@ -445,15 +445,15 @@ msgid "Change cut mode"
 
 msgid "Tolerance"
-msgstr "Допуск"
+msgstr "Зазор"
```
Технически тут речь о зазоре между частями при разрезе ласточкиным хвостом.

AndylgTom replied:
ок, раз одна строка и там и в «Добавить соединение» используется.

*Просто перевел по аналогии msgid "Orca Tolerance Test" - "Тест допусков от Orca

Felix14-v2 commented on localization/i18n/ru/OrcaSlicer_ru.po:
```diff
@@ -445,15 +445,15 @@ msgid "Change cut mode"
 # ??? короче Провести линию разреза
 msgid "Draw cut line"
-msgstr "Провести произвольно секущую плоскость"
+msgstr "Нарисовать проекцию разреза"
```
Ещё есть варианты `Разрезать линией`, `Нарисовать линию разреза`, но тут речь о проекции плоскости разреза.

AndylgTom replied:
Имхо, не... нарисовать так-то обще звучит с других оттенком, здесь Линия проводится и на основе ней получается разрез. Плюс, с местом проблем там нет, ибо окно само по себе широкое

Felix14-v2 replied:
А так?
```suggestion
msgstr "Провести разрез"
```

AndylgTom replied:
> А так?

Яж говорю, ты не разрез проводишь, разрез это результат. Выше же сам писал термин 'плоскость сечения', мне он кстати не нравится, чаще говорят  - секущая плоскость (можно в гугле проверить соотношение употребления фраз).

Felix14-v2 replied:
> чаще говорят - секущая плоскость

Во, действительно отличный вариант, и где-то уже даже используется.
```suggestion
msgstr "Разместить секущую плоскость"
```

Felix14-v2 commented on localization/i18n/ru/OrcaSlicer_ru.po:
```diff
@@ -471,10 +471,10 @@ msgid "Move connector"
 
 msgid "Add connector to selection"
-msgstr "Добавить соединение к выбранному"
+msgstr "Выделить соединение"
```
Упростил и сжал, смысл тут в контексте последующих строк не теряется:
<img width="451" height="256" alt="image" src="https://github.com/user-attachments/assets/f3ebb955-db08-4bc4-ba11-becd5cde1a59" />

Дальше упрощения/сжатия без потери смысла не буду отдельно комментировать, если что – комментарии приветствуются.

AndylgTom replied:
Абсолютно нормально.
Ты не выделяешь, выбираешь

Felix14-v2 replied:
Не понял – не подходит «Выделить ...»?
```suggestion
msgstr "Выбрать соединение"
```
Так тоже подходит вполне себе

Felix14-v2 committed:
Update OrcaSlicer_ru.po

Felix14-v2 commented on localization/i18n/ru/OrcaSlicer_ru.po:
```diff
@@ -786,10 +786,10 @@ msgid "Default font"
 
 msgid "The text cannot be written using the selected font. Please try choosing a different font."
-msgstr "Текст не может быть написан выбранным шрифтом. Пожалуйста, выберите другой шрифт."
+msgstr "Невозможно создать текст с выбранным шрифтом, попробуйте указать другой."
```
Тут и далее – исправление типовой ошибки перевода безличных предложений.

Felix14-v2 commented on localization/i18n/ru/OrcaSlicer_ru.po:
```diff
@@ -1047,7 +1047,7 @@ msgstr "Сброс вращения"
 # ??? Там в обе стороны же можно
 msgid "Rotate text Clock-wise."
-msgstr "Поворот текста на заданный угол."
+msgstr "Поворот текста в фронтальной плоскости (положительные значения соответствуют направлению часовой стрелки)."
```
Писал чуть ранее, добавляю исправление
```suggestion
msgstr "Наклон текста в фронтальной плоскости (положительные значения соответствуют направлению часовой стрелки)."
```

AndylgTom replied:
Так и так понятно, что вращение текста осуществляется относительно той плоскости, в которой текст был создан... И, по сути, и уточнение не нужно тут про направление, и так понятно, все знают, как стрелки часов ходят.

Felix14-v2 replied:
Вращение текста может быть свободным, если использовать инструмент вращения, наклон же – только в плоскости текста. Про связь направления и стрелки уберу, мне тоже оно кажется излишним (перевёл оригинальную строку).

Felix14-v2 commented on localization/i18n/ru/OrcaSlicer_ru.po:
```diff
@@ -1491,11 +1491,11 @@ msgstr ""
 # ?????6
 msgid "Process"
-msgstr "Профиль процесса"
+msgstr "Настройки"
```
Заголовок настроек печати. Да, эта история с "Process" (обработка печати) тянется ещё очень давно, но у нас `обработка` не прижилось, поэтому уж лучше оставить всем привычное `Настройки` [печати]. Ещё и немного места на боковой панели сэкономится.

AndylgTom replied:
"Профиль процесса" -  Тоже самое что у Пруше 'настройки печати'. Здесь же по другому 'настройки процесса печати', но это длинно, и они решили выкинуть последнее слово. А просто «настройки» уж точно не подходит
Но их терминологию Process я не собираюсь менять, по крайней мере пока.

Felix14-v2 replied:
Я собираюсь, потому что за 10 лет никто «Процессом» настройки печати так и не начал называть. Прямой перевод слова "Process" – это "обработка", а не "процесс" (который «повторяемая последовательность действий»). Это как раз пример совершенно неудачной локализации, которая тянется ещё со времён Slic3r. В контексте настроек «Профиль процесса» – это транслитерированная бессмыслица.

AndylgTom replied:
> Я собираюсь, потому что за 10 лет никто «Процессом» настройки печати так и не начал называть. Это как раз пример совершенно неудачной локализации, которая тянется ещё со времён Slic3r. В контексте настроек «Профиль процесса» – это транслитерированная бессмыслица.

Ну так предложи сначала это исправление авторам английского текста, это их терминология. И причем не просто «Настройка», а как правильно: «Настройки печати». Вопрос же не в этом, а в том, что все вики на англ. языке написаны. Плюс интерфейсы самих принтеров Бамбу на англ. языке написаны (кроме последнего A1). И никто там это править не будет. Можно написать и забить на это и само собой привыкнуть к этому диссонансу, я тебе просто суть проблемы говорю.

> Это как раз пример совершенно неудачной локализации, которая тянется ещё со времён Slic3r.

Путаешь, не было там такого никогда, так же как и в Пруше, в них это нормально названо. А пошло это все от Бамбуслайсера.

Felix14-v2 commented on localization/i18n/ru/OrcaSlicer_ru.po:
```diff
@@ -1491,11 +1491,11 @@ msgstr ""
 # ?????6
 msgid "Filament"
-msgstr "Профиль прутка"
+msgstr "Материал"
```
Профили – это разделом ниже, тут секция настроек катушек с материалом (добавление/удаление катушек).

Felix14-v2 commented on localization/i18n/ru/OrcaSlicer_ru.po:
```diff
@@ -2001,7 +2001,7 @@ msgid "Add Primitive"
 
 msgid "Add Handy models"
-msgstr "Добавьте тестовую модель"
+msgstr "Добавить тестовую модель"
```
Это не совет, а кнопка с действием

Felix14-v2 commented on localization/i18n/ru/OrcaSlicer_ru.po:
```diff
@@ -2139,22 +2139,22 @@ msgid "Right button click the icon to drop the object settings"
 
 msgid "Click the icon to reset all settings of the object"
-msgstr "Нажмите на значок, чтобы сбросить все настройки модели."
+msgstr "Сбросить настройки"
```
Тут и далее – сокращение совершенно бесполезной конструкции, которая колоссально усложняет чтение и не добавляет контекста. Подсказка и так выводится при наведении на кнопки. Значительно улучшает восприятие "с первого прочтения".

Felix14-v2 commented on localization/i18n/ru/OrcaSlicer_ru.po:
```diff
@@ -2791,7 +2791,7 @@ msgid "Install failed"
 
 msgid "Portions copyright"
-msgstr "С использованием разработок"
+msgstr "Сторонние компоненты"
```
Это не заголовок, а кнопка с вызовом списка сторонних разработок.
<img width="708" height="508" alt="image" src="https://github.com/user-attachments/assets/2b5ae60e-2eda-4e8a-9fa8-30090ea88518" />

Felix14-v2 commented on localization/i18n/ru/OrcaSlicer_ru.po:
```diff
@@ -2810,13 +2810,14 @@ msgstr "Orca Slicer основан на PrusaSlicer и BambuStudio"
 msgid "Libraries"
 msgstr "Библиотеки"
-
+# ??? Тут слайсер добавляет двоеточие
```
```suggestion
```

Felix14-v2 commented on localization/i18n/ru/OrcaSlicer_ru.po:
```diff
@@ -2810,13 +2810,14 @@ msgstr "Orca Slicer основан на PrusaSlicer и BambuStudio"
+# ??? Тут слайсер добавляет двоеточие
 msgid "This software uses open source components whose copyright and other proprietary rights belong to their respective owners"
-msgstr "В данном программном обеспечении используются компоненты с открытым исходным кодом, авторские и иные права на которые принадлежат их соответствующим владельцам."
+msgstr "В данном программном обеспечении используются компоненты с открытым исходным кодом, авторские и иные права на которые принадлежат их соответствующим владельцам"
```
В конце строки слайсер добавляет двоеточие
<img width="1105" height="113" alt="image" src="https://github.com/user-attachments/assets/893b2030-f6ec-4fbe-aa4c-34eddf1a8cde" />

AndylgTom replied:
Перефразировал.
В данном программном обеспечении используются компоненты с открытым исходным кодом, авторские и иные права на которые принадлежат следующим соответствующим владельцам

Felix14-v2 replied:
> следующим соответствующим

Логическая тавтология получается же – либо `следующим владельцам`, либо `соответствующим владельцам`

Felix14-v2 commented on localization/i18n/ru/OrcaSlicer_ru.po:
```diff
@@ -2810,13 +2810,14 @@ msgstr "Orca Slicer основан на PrusaSlicer и BambuStudio"
-msgstr "В данном программном обеспечении используются компоненты с открытым исходным кодом, авторские и иные права на которые принадлежат их соответствующим владельцам."
+msgstr "В данном программном обеспечении используются компоненты с открытым исходным кодом, авторские и иные права на которые принадлежат их соответствующим владельцам"
 
+# ??? Используется, судя по всему, только в паре с "OrcaSlicer", убираем пару гласных
```
```suggestion
```

Felix14-v2 commented on localization/i18n/ru/OrcaSlicer_ru.po:
```diff
@@ -2810,13 +2810,14 @@ msgstr "Orca Slicer основан на PrusaSlicer и BambuStudio"
 #, c-format, boost-format
 msgid "About %s"
-msgstr "О %s"
+msgstr "Об %s"
```
Используется в единственном месте, в заголовке окна
<img width="124" height="60" alt="image" src="https://github.com/user-attachments/assets/bced3f52-09ae-4c12-82a9-cf9714f3a4a5" />

AndylgTom replied:
Исправил "О программе %s" , так же как сам пункт меню называется

Felix14-v2 replied:
Поддерживаю
```suggestion
msgstr "О программе %s"
```

Felix14-v2 commented on localization/i18n/ru/OrcaSlicer_ru.po:
```diff
@@ -2977,7 +2978,7 @@ msgid "Enable AMS"
 
 msgid "Print with filaments in the AMS"
-msgstr "Печать пластиковыми нитями из АСПП."
+msgstr "Печать катушками из АСПП."
```
Вообще, я бы AMS-подобные решения локализовал иначе, слишком контринтуитивно звучит локализация аббревиатуры. Но пока просто улучшение восприятия смысла.

AndylgTom replied:
Прекрати пихать везде катушки, нет там такого слова 😁
Я б придерживался офиц. написания (если б хотели написать катушки, то написали бы) ибо если придираться к формулировкам... ты же не катушками печатаешь, а пластиком с этих катушек. Какое слово важнее то? 
Тогда уж типа - Печать пластиковыми нитями с катушек из АСПП.

Felix14-v2 replied:
> Прекрати пихать везде катушки, нет там такого слова 😁

Будет – в AMS у нас загружаются катушки, а не "пластиковые нити". И вообще, катушка хорошо воспринимается в местах, где "филамент", "материал", "пруток" или "пластиковая нить" звучит криво. Когда я набрасывал глоссарий, заметил, что `катушка` объединяет в себе всю эту неразбериху в одном термине. За отдельными исключениями (во многих местах по смыслу уместнее "материал"). Поэтому катушки спасут этот слайсер от несогласованности :D

> Я б придерживался офиц. написания

А где есть такое написание? Можно ссылку?

> если придираться к формулировкам... ты же не катушками печатаешь, а пластиком с этих катушек

Тогда лучше материал. `Пластиковая нить` – это вообще гомункул какой-то, оно семантически к филаменту относится даже хуже, чем `пруток`.

Felix14-v2 commented on localization/i18n/ru/OrcaSlicer_ru.po:
```diff
@@ -4033,7 +4034,7 @@ msgstr "Время"
 # ??? Загрузка
 msgid "Usage"
-msgstr "Использование"
+msgstr "Расход"
```
Окно просмотра свойств печати вообще надо делать настолько компактным, насколько возможно – на некоторых системах из-за длинных строк оно половину экрана закрывает, что делает использование слайсера крайне некомфортным.
<img width="531" height="665" alt="image" src="https://github.com/user-attachments/assets/7897ee26-1620-4b49-bc89-541c4a693d89" />

Далее аналогично.

AndylgTom replied:
Чтобы проблем не было надо иметь монитор 21:9 😃

Felix14-v2 commented on localization/i18n/ru/OrcaSlicer_ru.po:
```diff
@@ -4127,11 +4128,12 @@ msgstr "Переключиться на обычный режим"
+# ??? Кнопка действия
 msgid "Adaptive"
-msgstr "Адаптивная"
+msgstr "Адаптировать"
```
Тут это кнопка действия (переменная высота слоёв → адаптировать высоту к кривизне поверхности)
<img width="534" height="220" alt="image" src="https://github.com/user-attachments/assets/baf6f900-12fe-4e6f-b27e-4dd23e666c37" />

Felix14-v2 commented on localization/i18n/ru/OrcaSlicer_ru.po:
```diff
@@ -4140,25 +4142,26 @@ msgid "Radius"
 
 msgid "Keep min"
-msgstr "Сохранять минимумы"
+msgstr "Не смещать минимумы"
```
Минимумы в любом случае сохраняются, лучше указать конкретно на то, что делает эта опция – блокирует перемещение точек с минимальной высотой слоя на графике высоты слоя при работе сглаживания.

AndylgTom replied:
Ну тут смотря какая логика у кого.
Не смещать минимумы - так их никто и не смещает по факту, они просто сглаживаются при нажатии кнопки «Сгладить». (точки минимумов сохраняются)
Тут уж тогда уместней написать «Не сглаживать минимумы», чтоб как-то связать с кнопкой «Сгладить», поэтому вполне уместно так же, как на англ. языке, оставить.

Felix14-v2 replied:
> Не смещать минимумы - так их никто и не смещает по факту, они просто сглаживаются при нажатии кнопки «Сгладить».

В том-то и дело, что смещаются:

https://github.com/user-attachments/assets/5fad215a-e20f-423e-addb-0c5f2742d15a

> Тут уж тогда уместней написать «Не сглаживать минимумы», чтоб как-то связать с кнопкой «Сгладить»

В том-то и дело, что эта галочка не влияет на их сглаживание (см. выше)

AndylgTom replied:
> В том-то и дело, что эта галочка не влияет на их сглаживание (см. выше)

Ну как не влияет то, на видео видно же, что без галочки минимумы и сглаживаются. С галочкой нет.

Felix14-v2 replied:
Имел ввиду, что сглаживание минимумов всё равно происходит, просто сами они при этом не перемещаются – сглаживается окружающая их область.

Felix14-v2 commented on localization/i18n/ru/OrcaSlicer_ru.po:
```diff
@@ -4140,25 +4142,26 @@ msgid "Radius"
+# ??? Неточность в исходной строке: детализация зависит от многих факторов, тут же меняется исключительно высота слоя
 msgid "Add detail"
-msgstr "Увеличить детализацию"
+msgstr "Уменьшить высоту слоя"
```
Неточность в исходной строке: детализация зависит от многих факторов, тут же меняется исключительно высота слоя. Аналогично в строке №4158

Felix14-v2 commented on localization/i18n/ru/OrcaSlicer_ru.po:
```diff
@@ -4140,25 +4142,26 @@ msgid "Radius"
 msgid "Left mouse button:"
 msgstr "Левая кнопка мыши:"
 
+# ??? Неточность в исходной строке: детализация зависит от многих факторов, тут же меняется исключительно высота слоя
```
```suggestion
```

Felix14-v2 commented on localization/i18n/ru/OrcaSlicer_ru.po:
```diff
@@ -4140,25 +4142,26 @@ msgid "Radius"
 
 msgid "Reset to base"
-msgstr "Сброс до базовой высоты слоя"
+msgstr "Вернуть к базовой высоте слоя"
```
Тут инструмент работает не как моментальный сброс, а как постепенный возврат к целевому значению.

Felix14-v2 commented on localization/i18n/ru/OrcaSlicer_ru.po:
```diff
@@ -4167,25 +4170,25 @@ msgid "Smoothing"
 
 msgid "Mouse wheel:"
-msgstr "Колесо мыши:"
+msgstr "Вращение колеса мыши:"
```
Можно подумать, что нужно нажать СКМ, а не крутить колёсико. Возможно, стоит сократить до `Вращение СКМ` с целью экономии пространства, но лучше по возможности избегать аббревиатур.

Felix14-v2 commented on localization/i18n/ru/OrcaSlicer_ru.po:
```diff
@@ -4283,10 +4286,10 @@ msgid "Paint Toolbar"
 
 msgid "Explosion Ratio"
-msgstr "Коэффициент разброса"
+msgstr "Разнесение"
```
Взято на основе локализации КОМПАС-3D

AndylgTom replied:
А как быть что это коэффициент, а не миллиметры как в компасе? А то вдруг кто-будет вбивать значения думая что там мм. Важно-неважно это?

*вариант, Разнос компонентов

Felix14-v2 replied:
Хороший вопрос, я об этом не думал. Но мне кажется, не принципиально, т.к. коэффициент там очень чувствительный, при значении `3` части модели разлетаются очень далеко, явно не на 3мм. А при нулевом разнесении изначально стоит значение `1`.

Felix14-v2 commented on localization/i18n/ru/OrcaSlicer_ru.po:
```diff
@@ -4283,10 +4286,10 @@ msgid "Paint Toolbar"
 
 msgid "Section View"
-msgstr "Отсечение вида"
+msgstr "Сечение"
```
Аналогично строке №55, просто заглавная в оригинале вместо строчной

Felix14-v2 commented on localization/i18n/ru/OrcaSlicer_ru.po:
```diff
@@ -4432,7 +4435,7 @@ msgid "Logging"
 
 msgid "Preview"
-msgstr "Предпросмотр нарезки"
+msgstr "Просмотр нарезки"
```
Мелочь – а текст теперь не вылезает за границы дозволенного:
<img width="311" height="296" alt="image" src="https://github.com/user-attachments/assets/3e78e04a-da73-441c-a543-f181eebbf4ed" />

AndylgTom replied:
А панель инструментов не смущает? :) Это не мы должны править, а они.

Felix14-v2 commented on localization/i18n/ru/OrcaSlicer_ru.po:
```diff
@@ -4628,7 +4631,7 @@ msgid "Export"
 
 msgid "Quit"
-msgstr "Выйти"
+msgstr "Выход"
```
В целом, в интерфейсе большинства системных программ используется `Выход`, да и с соседними кнопками лучше согласуется.

Felix14-v2 commented on localization/i18n/ru/OrcaSlicer_ru.po:
```diff
@@ -4747,29 +4750,30 @@ msgstr "Калибровка температуры"
 msgid "Pass 1"
 msgstr "Проход 1"
 
+# ??? У коэффициента не может быть скорости
```
```suggestion
```

Felix14-v2 commented on localization/i18n/ru/OrcaSlicer_ru.po:
```diff
@@ -4747,29 +4750,30 @@ msgstr "Калибровка температуры"
+# ??? У коэффициента не может быть скорости
 msgid "Flow rate test - Pass 1"
-msgstr "Тест скорости потока - 1-ый проход"
+msgstr "Тест потока - 1-ый проход"
```
У коэффициента не может быть скорости (исправлено тут и дальше). Задно строка стала чуть короче.

AndylgTom replied:
Может...
А как это будет согласовываться с 
msgid "Flow Dynamics" 
msgstr "Динамика потока"
аля Pressure Advance - K-Factor
?
в том смысле, что не то что бы не правильно, а просто они четко были различимы смыслово и визуально две эти калибровки в интерфейсе бамбу.
Т.е пишется
"Flow Dynamics" - "Динамика потока" и "Flow Rate" - "Скорость потока"

писать в одно слово что-ли теперь там "Flow Rate" - "Поток"?

Felix14-v2 replied:
> они четко были различимы смыслово и визуально две эти калибровки в интерфейсе бамбу.
Т.е пишется
"Flow Dynamics" - "Динамика потока" и "Flow Rate" - "Скорость потока"

Если я правильно понимаю, т.н. `Динамика потока` – это просто LA/PA. На русский язык эти термины не привычно переводить, хотя `Контроль давления` на мой взгляд довольно близко передаёт суть оригинального `Pressure Advance`, я бы его предложил на роль перевода для всех трёх терминов (Linear Advance, Pressure Advance и Flow Dynamics).

Felix14-v2 commented on localization/i18n/ru/OrcaSlicer_ru.po:
```diff
@@ -4799,7 +4803,7 @@ msgid "Input Shaping"
 
 msgid "VFA"
-msgstr "Тест на вертикальные артефакты (VFA)"
+msgstr "Тест ряби"
```
Традиционно эти артефакты называют "рябью", не вижу смысла в таком нагромождении.

AndylgTom replied:
так уточнение же важно, а то под рябью хоть что можно подразумевать. Или писать в скобках (VFA с расшифровкой)
плюс Тест ряби звучит коца как-то,... Тест на выявление ряби -> Тест на рябь получше

Felix14-v2 replied:
В обсуждении к строке №7696 уже [упоминал](https://github.com/AndylgTom/OrcaSlicer/pull/1/files?diff=split&w=0#r2358751597), что можно спокойно обойтись и без уточнения, в русскоязычном сегменте интернета рябь хорошо гуглится. Про уточнение в скобках – идея хорошая, но лучше тогда в заголовке окна теста.

> Тест ряби звучит коца как-то

Да нет, вполне нормально. Поиск по чату K3D могу привести в пример, почти все так и пишут:
<img width="523" height="1251" alt="image" src="https://github.com/user-attachments/assets/f28905ad-a8c7-423b-8405-82f95398f1d1" />

Felix14-v2 commented on localization/i18n/ru/OrcaSlicer_ru.po:
```diff
@@ -5726,10 +5730,10 @@ msgid "Objects"
 
 msgid "Advance"
-msgstr "Расширенный"
+msgstr "Расширенные"
```
Согласование с `Настройки`

AndylgTom replied:
У меня другая логика - Относиться к режиму отображения интерфейса Расширенный, как в Пруще было 3 штуки, туп по факту тоже 3, еще Режим разработчика есть

Felix14-v2 replied:
Ну, одно дело согласовывать терминологию, это дело хорошее. И совсем другое – отсылать на элементы оформления в другом слайсере.

Felix14-v2 commented on localization/i18n/ru/OrcaSlicer_ru.po:
```diff
@@ -5726,10 +5730,10 @@ msgid "Objects"
 
 msgid "Compare presets"
-msgstr "Сравнение профилей"
+msgstr "Сравнить профили"
```
Продолжается история с кнопками действий

Felix14-v2 commented on localization/i18n/ru/OrcaSlicer_ru.po:
```diff
@@ -5803,16 +5807,16 @@ msgid "Flushing volumes"
 
 msgid "Add one filament"
-msgstr "Добавить пластиковую нить"
+msgstr "Добавить катушку"
```
<img width="583" height="71" alt="image" src="https://github.com/user-attachments/assets/a367a7c5-1871-4b67-9099-2d4e67a7fe39" />

AndylgTom replied:
Опять же, не хочу я использовать катушку там, где нет этого слова. Вот спросить у них, почему они так написали, они наверняка ответят что-то типа, что по факту добавляешь профиль материала катушки, а не катушку. 
Это, наверное, просто совпадение, но на иконке выделен цветом именно пруток на этой катушке. 

В бамбу там просто + и -
А в Пруше по другому, при добавлении экструдеров появляются столько же профилей прутков.

ХЗ, хз.

Felix14-v2 replied:
Тут проблема в том, что `Добавить материал` пользователь может ошибочно воспринять как добавление материала из библиотеки, а не добавление "экземпляра" материала в печать.

В конце концов, после добавления можно использовать один и тот же материал, просто печатать из двух разных катушек (без понятия, зачем, но такая возможность есть):
<img width="746" height="180" alt="{93EECFC3-709C-425F-AC8D-D340F4DB53B6}" src="https://github.com/user-attachments/assets/41c84cad-a211-4583-be1a-a3c23e4ed074" />
> без понятия, зачем

Хотя как же – как раз для печати разными цветами одного и того же пластика как раз

Felix14-v2 commented on localization/i18n/ru/OrcaSlicer_ru.po:
```diff
@@ -5803,16 +5807,16 @@ msgid "Flushing volumes"
 
 msgid "Set filaments to use"
-msgstr "Выбор пластиковой нити"
+msgstr "Выбрать материал"
```
Возможно, стоит заменить на что-то вроде
```suggestion
msgstr "Настроить список материалов"
```

AndylgTom replied:
Да нормально - "Выбрать материал" или "Выбрать материалы"

Или развернуто - Выбрать профили материалов

Felix14-v2 commented on localization/i18n/ru/OrcaSlicer_ru.po:
```diff
@@ -6423,13 +6427,13 @@ msgid "Disable to use latest network plugin that supports new BambuLab firmwares
 
 msgid "Imperial"
-msgstr "Английская СИ"
+msgstr "Имперская СИ"
```
Субъективно классический вариант перевода встречаю чаще, возможно стоит оставить как есть.

AndylgTom replied:
Уже менял несколько раз туда сюда, всем не угодишь, у Имперская двоякий смысл
Варианты
"Metric" - Метрическая система, Метрическая система мер, Метрические единицы

"Imperial" - Имперская или Английская система, Имперская или Английская система мер,  Имперские или Английские единицы

Felix14-v2 replied:
Сам с ней почти не сталкиваюсь, поэтому первая мысль была, что неправильно называть её "английской", т.к. даже в Англии уже в отдельных сферах метрическая СИ является обязательной, а привязывать СИ к английскому языку как-то неправильно. `Имперская СИ` же семантически отсылает к Британской Империи, которая ныне уже не существует, но её СИ всё ещё в обиходе.

Честно говоря, изменение не принципиально. С одной стороны, `Имперская СИ` воспринимается более правильно, с другой – человеку, не знакомому с историей, `Английская СИ` будет понятнее. Не знаю.

Felix14-v2 commented on localization/i18n/ru/OrcaSlicer_ru.po:
```diff
@@ -6943,7 +6947,7 @@ msgid "Task canceled"
 
 msgid "Bambu Cool Plate"
-msgstr "Не нагреваемая пластина Bambu"
+msgstr "Низкотемпературная пластина Bambu"
```
Греть-то его можно, просто рабочий диапазон температур ниже.
```suggestion
msgstr "Низкотемпературный лист Bambu"
```

AndylgTom replied:
Угу, Низкотемпературная в голову не пришло.

Но...
Понятно что 'лист' так короче, понятно что есть понятие 'лист метала'... но так же есть 'лист в тетради', 'лист с дерева'. Слишком много понятий в слове 'лист' + это что-то однородное, а тут так сказать больше конкретики, тонкая пластина, 'пирог' разных материалов (покрытие на метал. основе).

И опятьже как тогда в предложениях писать Plate? Например:
Автоориентация всех моделей на текущей печатной пластине  - типа Автоориентация всех моделей на текущей печатном листе?
Температура стола для первого слоя. 0 означает, что пластиковая нить не поддерживает печать на этой печатной пластине. - типа Температура стола для первого слоя. 0 означает, что пластиковая нить не поддерживает печать на этом печатном листе.
... как-то не очень звучит "печатном листе"

Хз, смысла не вижу шило на мыло менять

Felix14-v2 replied:
>Этот случай очень простой и понятный:
<img width="326" height="97" alt="{15AA2408-8F52-45F0-A495-9DCBE6CBD62E}" src="https://github.com/user-attachments/assets/8a9d988a-002a-4136-9c23-df1603878ef4" />

> Вообще, упомянутая строка – это ярчайший пример того, как не надо переводить (я почти уверен, что её переводила машина).

Вот как раз смысл-то не в этом. По твоему варианту получается, как будто модель где-то висит в воздухе, и при нажатии кнопки она просто ложится на стол как есть. А смысл в том, что при нажатии выбирается плоскость с наибольшей площадью, и этой стороной она ложится на стол. И делается это в автоматическом режиме, в отличие от другого инструмента, где ты уже сам можешь сторону выбрать. Т. е. твой вариант слишком краток для полного понимания сути.

Felix14-v2 commented on localization/i18n/ru/OrcaSlicer_ru.po:
```diff
@@ -7233,7 +7237,7 @@ msgstr "Не удалось выйти."
 #, c-format, boost-format
 msgid "Save current %s"
-msgstr "Сохранить текущий %s"
+msgstr "Сохранить %s"
```
Лучше всё-таки без согласования с родом.

AndylgTom replied:
не помню, где это?

Felix14-v2 replied:
Это везде, где есть диалог сохранения. Настройки, материал и т.д.

Felix14-v2 commented on localization/i18n/ru/OrcaSlicer_ru.po:
```diff
@@ -7497,13 +7501,13 @@ msgid "Reserved keywords found"
 
 msgid "Setting Overrides"
-msgstr "Переопределение параметров"
+msgstr "Замена настроек"
```
Тут и далее – оптимизация названий вкладок, чтобы уместить все в ширину окна:
<img width="885" height="155" alt="image" src="https://github.com/user-attachments/assets/3c91bacf-7161-4ae1-8e3e-8913fe856a4f" />

AndylgTom replied:
Переопределение вполне устоявшийся, универсальный и технически понятный вариант.
Яб сказал ты значения переопределяешь, но полностью и развернуто тогда было бы типа «Переопределение значений параметров», что получается слишком длинно. Если просто писать «Переопределение» то как-то куце по смыслу.

Felix14-v2 replied:
Ох, эта строчка... Я прошерстил общий глоссарий Microsoft по всем их программам, чтобы посмотреть, какие существуют варианты компактного перевода (компактность здесь важна, поскольку одна из крупнейших проблем локализации по части UX – это необходимость совершать лишние действия для перемещения по вкладкам из-за того, что суммарная длина вкладок превышает ширину окна).

Переопределение – это больше кодерский термин, ноги тут тянутся из «определение переменной»/«переопределение значения» (define variable/override value). "В быту" же для обычных пользователей существуют такие варианты:
- перекрытие
- замещение
- замена
- корректировка
- перезапись

В целом, почти всё с натяжкой подходит и к нашему случаю, но `замена` на мой взгляд – и самый компактный, и самый уместный по контексту. У нас ведь есть обычные настройки принтера, которые мы задаём в соотв. разделе (это define), а есть настройки, которые мы меняем в зависимости от выбранного материала (это override). Эти настройки задаются в профиле материала, и они замещают (заменяют) собой основные (те, что идут по умолчанию). Собственно, поэтому и предложил назвать раздел `Замена настроек`.

AndylgTom replied:
Да это не то окно, где это настолько важно. Вот боковое окно куда важнее.
Насчет бокового окна мне подход у Creality print понравился, там эти вкладки сделаны сбоку в виде иконок, при наведении на которые уже надпись появляется, может когда-нибудь и тут додумаются до этого.

Felix14-v2 replied:
> это не то окно, где это настолько важно. Вот боковое окно куда важнее.

А разве эта строчка используется ещё где-то кроме названия вкладок?
<img width="883" height="214" alt="{53919CDD-C2D7-4130-972A-946A38A56D04}" src="https://github.com/user-attachments/assets/188979be-9e97-4df8-a64c-a654b2201472" />

<img width="1700" height="788" alt="{F1CB69BD-6839-4BC3-83FE-12BDA37B2DDE}" src="https://github.com/user-attachments/assets/bc12ec93-86b3-4235-9e9a-358f61165046" />

Felix14-v2 commented on localization/i18n/ru/OrcaSlicer_ru.po:
```diff
@@ -7685,17 +7689,17 @@ msgid "Template Custom G-code"
 
 msgid "Resonance Avoidance"
-msgstr "Предотвращение резонанса"
+msgstr "Борьба с рябью"
```
Тут, строго говоря, снова ошибка в исходной строке. Рябь, которую эта настройка избегает, не имеет ничего общего с резонансом (тестом Input Shaping). В любом случае, так и компактнее, и корректнее.

AndylgTom replied:
Борьба, ну такое... что-то странное...
Я вот так все-таки думаю.
Заголовок написать более развернуто с упоминанием VFA, а саму опцию с галочкой «Предотвращать рябь»

<img width="742" height="187" alt="image" src="https://github.com/user-attachments/assets/6ba74a1d-7211-40d2-8e2b-28e5bd4ba169" />

плюс наверное лучше привести в единообразию название параметра "Min" и "Max" также как в Тесте на рябь.... или там изменить Мин. скорость, Макс. скорость

Felix14-v2 replied:
> Заголовок написать более развернуто с упоминанием VFA

Зачем? В русскоязычном сообществе нет необходимости в употреблении `VFA`, потому что эта аббревиатура эффективно заменяется на обычную и понятную всем `рябь`. Слово отлично передаёт суть дефекта и не нуждается в сокращении англоязычного термина.

> саму опцию с галочкой «Предотвращать рябь»

`Предотвращать` здесь некорректно, т.к. семантика слова "Предотвращать" – исправлять проблему или её источник, здесь же мы именно что избегаем условий её возникновения за счёт снижения скорости печати. А `борьба` неплохо обобщает вообще всю суть раздела, при этом сохраняя лаконичность (краткость без потери смысла упрощает восприятие):
<img width="844" height="134" alt="{E5875EA8-F0DB-4DC7-A599-8B988125E5DE}" src="https://github.com/user-attachments/assets/f7fab9a9-cf9e-4125-bfbb-d13298b0371b" />

AndylgTom replied:
> Зачем? 
Чтобы люди ища информацию в интернете понимали, что такое в принципе рябь соотнося ее с VFA. Вот открывают они руководство и там про "рябь" ни слова, зато про VFA написано, все руководства и вики на английском же.

Felix14-v2 replied:
>Чтобы люди ища информацию в интернете понимали, что такое в принципе рябь соотнося ее с VFA.

Не думаю, что это будет проблемой: поисковики Google, Yandex, Bing (да и другие тоже) используют локализованную выдачу результатов:
| Google | Yandex |
|:-:|:-:|
| <img width="1080" height="1860" alt="image" src="https://github.com/user-attachments/assets/a13bd89a-8d82-40ce-a5ae-33fd737ff2d1" /> | <img width="1080" height="1860" alt="image" src="https://github.com/user-attachments/assets/6bc7e571-b284-46ff-b73c-50dfc8b5be5c" /> |

А вот если явно написать `VFA`, тогда да, поисковики выдают англоязычные ресурсы (хотя даже так на первой строке в выдаче всё равно висит [статья Дмитрия](https://k3d.tech/calibrations/vfa/)). В целом, я бы упомянул `Vertical Fine Artifacts` в подсказке, там у нас место не ограничено, и как раз можно сослаться на англоязычное название термина. Или в заголовке теста, там тоже места хватает. А лучше и там, и там.

Felix14-v2 commented on localization/i18n/ru/OrcaSlicer_ru.po:
```diff
@@ -7706,11 +7710,12 @@ msgstr "Максимальные ускорения"
+# ??? Ошибка в исходной строке (раздел посвящён общей настройке комбинированной печати, в т.ч. двумя отдельными фидерами)
 msgid "Single extruder multi-material setup"
-msgstr "Параметры одиночного ММ экструдера"
+msgstr "Конфигурация комбинированной печати"
```
"Конфигурация" тут не случайно – в секции настраивается аппаратная часть принтера, причём в том числе и количество экструдеров. Так что и в исходной строке есть неточность.

AndylgTom replied:
Конфигурация - ок


Уверен, что люди поймут, что такое комбинированная печать, и не будут что там в голове левое додумывать? Где это будешь объяснять? 
По-моему, из всех утюгов звучит мультиматериальная печать, и все понимают, о чем речь. Также мультиматериальная удобно сокращать ММ — ММ печать, ММ экструдер, чтобы голову не ломать, как сократить. Или использовать тогда оба варианта?

Пока не знаю, в первом варианте/этапе пока оставлю как есть. И так много правок делать сразу в трех слайсерах. На втором этапе после обновления репозиториев можно подумать и поправить... или чуть позже.

Felix14-v2 replied:
> Уверен, что люди поймут, что такое комбинированная печать, и не будут что там в голове левое додумывать? Где это будешь объяснять?

> мультиматериальная удобно сокращать ММ — ММ печать, ММ экструдер, чтобы голову не ломать, как сократить

`Экструдер ММ` здесь воспринимается ещё хуже. Аббревиатур и сокращений надо избегать везде, где только можно, потому что главная задача локализации – обеспечить понимание пользователем семантики каждой строчки с первого прочтения. Если это условие не выполняется, значит локализация плохая. А аббревиатуры ни с первого, ни со второго прочтения понять невозможно, если не дать в подсказке к кнопке комментарий, а в случае с разделом настроек это невозможно.
Комбинированная печать по семантике относит нас... ну... к комбинированной печати несколькими материалами, всё содержание раздела об этом кричит. Был ещё вариант с `композитной печатью`, однако это конфликтует по смыслу с печатью композитными материалами (угле/стеклонаполненные), поэтому такой вариант здесь наиболее уместный. Передаёт ту же семантику комбинирования сочетаний материалов в одной детали, однако на уровне уже структуры детали, а не материала.

> из всех утюгов звучит мультиматериальная печать, и все понимают, о чем речь

Звучит это так из-за изначально плохой локализации. Я переводчеством занимаюсь уже много лет, и наглядно вижу, как меняется разговорная терминология на новую, если она более удачна, чем старая. Один из первых проектов, которые я переводил коллективно, имел несколько строк, которые впоследствии заменили из-за требований автора программы. И люди в разговорной речи продолжили использовать (да и по сей день, 8 лет спустя используют) старый устоявшийся вариант, потому что он тупо удобнее и понятнее.

Поэтому не стоит держаться за старый вариант, особенно если он создаёт проблемы с пониманием. А нынешний, судя по чату K3D, ещё как создаёт.

Felix14-v2 commented on localization/i18n/ru/OrcaSlicer_ru.po:
```diff
@@ -10218,7 +10223,7 @@ msgstr ""
 
 msgid "Shrinkage (XY)"
-msgstr "Компенсация усадки по XY"
+msgstr "Усадка по XY"
```
Тут задаётся именно измеренная усадка (значение меньше 100%), а не её компенсация (обратная величина будет больше 100%). Аналогично строке №10237

Felix14-v2 commented on resources/web/data/text.js:
```diff
@@ -1045,23 +1045,23 @@ var LangText = {
     t39: "Принять",
-    t40: "Сеть отключена. Пожалуйста, проверьте подключение и попробуйте снова.",
-    t47: "Пожалуйста, выберите регион входа",
+    t40: "Сеть отключена. Проверьте подключение и попробуйте снова.",
```
Традиционное "Пожалуйста" в английском языке не принято переводить в контексте обязательных действий, когда выбора пользователю не даётся.

Felix14-v2 commented on resources/web/data/text.js:
```diff
@@ -1111,11 +1111,11 @@ var LangText = {
     orca2: "Информации о модели отсутствует",
     orca3: "Режим конфиденциальности",
-    orca4: "Это остановит передачу данных в облачные сервисы Bambu. Пользователи, которые не используют принтеры Bambu Lab или используют режим «Только LAN», могут безопасно включить эту функцию.",
+    orca4: "Это остановит передачу данных в облачные сервисы Bambu. Пользователи, которые не используют принтеры Bambu Lab или используют их офлайн, могут спокойно включать эту опцию.",
```
Такой ещё есть вариант:
```suggestion
    orca4: "Это остановит передачу данных в облачные сервисы Bambu. Пользователи, которые не используют принтеры Bambu Lab или используют их локально, могут спокойно включать эту опцию.",
```

AndylgTom replied:
Насчет text.js я практики все принял, поэтому просто заменил его, чтобы тут не тыкать. Неудобно все же в браузере 😠

Кроме последнего...

— Это остановит передачу данных в облачные сервисы Bambu. Пользователи, которые не используют принтеры Bambu Lab или используют их в режиме «Только LAN», могут безопасно включить эту опцию.

Просто там пункт в меню принтера на английском так и называется «LAN only», и чтобы людей не путать, и он сразу, так сказать, в глаза бросался и сопоставлялся с русским переводом, поэтому так и писал везде. Опять же, не все понимают, что такое офлайн, некоторые будут думать, что и работа по локальной сеть нафиг отрубится при включении ползунка.

И вот потом надо глянуть, как его перевели в официальном переводе на модели A1 с учетом, что там маленький экран и надо впихнуть невпихуемое, не как на других моделях (где и интерфейс другой)... хотя хз.
<img width="582" height="424" alt="image" src="https://github.com/user-attachments/assets/735ec527-217a-4044-8353-5144dacf69de" />

Felix14-v2 replied:
Да, я поэтому в этом комментарии предложил ещё общий вариант с "используют принтер локально". Ещё есть такой инвертированный вариант:
```suggestion
    orca4: "Это остановит передачу данных в облачные сервисы Bambu. Помешает только владельцам Bambu Lab, не использующим режим «Только LAN».",
```

Кстати, изменения в те же файлы в репозитории лучше не вносить, потом будет проблемно объединять. Лучше сюда закидывать.

Felix14-v2 reviewed (commented):
Начинаем обсуждение :)

Felix14-v2 committed:
Update OrcaSlicer_ru.po

Felix14-v2 committed:
Update OrcaSlicer_ru.po

Felix14-v2 committed:
Update OrcaSlicer_ru.po

Felix14-v2 committed:
Update OrcaSlicer_ru.po

Felix14-v2 committed:
Merge branch 'main' of https://github.com/Felix14-v2/Andy-OrcaSlicer

Felix14-v2 committed:
Update OrcaSlicer_ru.po

Felix14-v2 committed:
Remove unnecessary comment

Felix14-v2 committed:
Update OrcaSlicer_ru.po

Felix14-v2 committed:
Update localization/i18n/ru/OrcaSlicer_ru.po

Felix14-v2 committed:
Merge branch 'main' of https://github.com/Felix14-v2/Andy-OrcaSlicer

AndylgTom commented on localization/i18n/ru/OrcaSlicer_ru.po:
```diff
@@ -309,22 +308,22 @@ msgid "Reset Rotation"
 
 msgid "Reset current rotation to real zeros."
-msgstr "Сброс текущего вращения на нулевое значение."
+msgstr "Сбросить ориентацию"
```
```suggestion
msgstr "Сбросить ориентацию на нулевое значение"
```

Felix14-v2 committed:
Update OrcaSlicer_ru.po

AndylgTom commented on localization/i18n/ru/OrcaSlicer_ru.po:
```diff
@@ -225,9 +225,8 @@ msgstr "Удаление окрашенного участка"
-# ??? Процедура переназначения материала завершена, выполнена
 msgid "Filament remapping finished."
-msgstr "Переназначение материала завершено."
+msgstr "Катушки переназначены."
```
```suggestion
msgstr "Материал успешно переназначен."
```

AndylgTom replied:
```suggestion
msgstr "Материал успешно заменён."
```

AndylgTom commented on localization/i18n/ru/OrcaSlicer_ru.po:
```diff
@@ -309,22 +308,22 @@ msgid "Reset Rotation"
 
 msgid "Reset current rotation to the value when open the rotation tool."
-msgstr "Сброс текущего вращения до значения, установленного при открытии инструмента вращения."
+msgstr "Сбросить ориентацию до предыдущих значений"
```
```suggestion
msgstr "Сбросить ориентацию к положению, установленному при открытии инструмента вращения."
```

AndylgTom commented on localization/i18n/ru/OrcaSlicer_ru.po:
```diff
@@ -414,7 +413,7 @@ msgstr "Угол скоса"
 
 # ??? отображается в инструменте Текст
```
```suggestion
# ??? в орке видать исправили
```

Felix14-v2 committed:
Update OrcaSlicer_ru.po

Felix14-v2 committed:
Update OrcaSlicer_ru.po

Felix14-v2 committed:
Update OrcaSlicer_ru.po

AndylgTom commented on localization/i18n/ru/OrcaSlicer_ru.po:
```diff
@@ -423,17 +422,17 @@ msgid ""
-"Двигайте, чтобы переместить секущую плоскость.\n"
-"Для переназначения стороны части модели используйте правую кнопку мыши."
+"ЛКМ – смена направления сечения.\n"
+"Перетаскивание – сдвиг плоскости сечения.\n"
```
секущая плоскость правильнее, ниже упомянул про это

AndylgTom commented on localization/i18n/ru/OrcaSlicer_ru.po:
```diff
@@ -599,7 +598,7 @@ msgid "Cut by Plane"
 
 msgid "non-manifold edges be caused by cut tool, do you want to fix it now?"
-msgstr "При работе режущего инструмента образовались открытые рёбра. Хотите починить это сейчас?"
+msgstr "При разделении образовались открытые рёбра. Попытаться исправить модель?"
```
```suggestion
msgstr "При разрезании образовались открытые рёбра. Хотите починить это сейчас?"
```

AndylgTom replied:
```suggestion
msgstr "При разрезании образовались открытые рёбра. Хотите починить их сейчас?"
```

AndylgTom commented on localization/i18n/ru/OrcaSlicer_ru.po:
```diff
@@ -786,10 +785,10 @@ msgid "Default font"
 
 msgid "Advanced"
-msgstr "Дополнительно"
+msgstr "Прочее"
```
Точно лучше так?

Относится в другим\прочим\дополнительным опциям\функциям в разных вкладках

Felix14-v2 replied:
Будь у орки нормальная система вкладок – да, «Расширенные» бы подошло идеально. Впрочем, менял я это только ради экономии места в настройках материала, а сейчас там за счёт других оптимизаций места стало хватать:
<img width="1258" height="232" alt="{103B8464-B245-4D8C-B7E0-3AB305696E40}" src="https://github.com/user-attachments/assets/a3bcccb7-d16f-46f5-b033-0117bda0b7ad" />

Так что да, верну на `Расширенные`

Felix14-v2 committed:
Merge branch 'main' into main

AndylgTom commented on localization/i18n/ru/OrcaSlicer_ru.po:
```diff
@@ -1091,13 +1090,13 @@ msgstr "Размер"
 #. TRN - Input label. Be short as possible
 msgid "Use surface"
-msgstr "Только на поверхности"
+msgstr "Проецировать"
```
Я всегда за развернутость :)

Предлагаю
msgid "Use surface"
msgstr "Проецировать на поверхность" или так уж и быть "Проецировать"

но ниже тогда
msgid "Per glyph"
msgstr "Проецировать с обтеканием"

ибо думаешь "Нормально к поверхности" более понятнее чем "Ориентация по глифу"? Сомневаюсь. А так конкретнее уже некуда

Felix14-v2 replied:
> Предлагаю msgid "Use surface" msgstr "Проецировать на поверхность" или так уж и быть "Проецировать"
> 
> но ниже тогда msgid "Per glyph" msgstr "Проецировать с обтеканием"
> 
> ибо думаешь "Нормально к поверхности" более понятнее чем "Ориентация по глифу"? Сомневаюсь. А так конкретнее уже некуда

(К строке №1093)


> Ориентация по глифу

Это не совсем корректно, так как символы не просто меняют свою ориентацию, они именно деформируются нормально к поверхности. Я бы оставил `Перпендикулярно к поверхности`, но не хотелось бы ширину окна увеличивать (`Проецировать на поверхность` это тоже касается). Разница небольшая, конечно, но заметная:

<img width="2560" height="1600" alt="Снимок экрана (246)" src="https://github.com/user-attachments/assets/c2001728-ddff-4eee-bf29-b1237ba0d5b6" />
<img width="2560" height="1600" alt="Снимок экрана (247)" src="https://github.com/user-attachments/assets/bab0da0e-699b-4058-b731-b5ccf24855e0" />

Но вообще, можно просто не париться и сделать как в оригинале – `По буквам`/`По символам`/`Посимвольно`/`Побуквенно`:
<img width="2560" height="1516" alt="image" src="https://github.com/user-attachments/assets/ed44de01-db2a-4e17-94e5-ca494cd3f69e" />

Так даже правильнее будет, поскольку в размещении текста на поверхности по буквам при отключённом проецировании непосредственно проецирования не происходит, это немного заметно на скриншоте выше. Для сравнения:
<img width="2560" height="1516" alt="image" src="https://github.com/user-attachments/assets/522a7feb-c62f-44c3-98e5-79a071f50c61" />

AndylgTom commented on localization/i18n/ru/OrcaSlicer_ru.po:
```diff
@@ -1184,7 +1183,7 @@ msgid "Undefined stroke type"
 
 msgid "Path can't be healed from self-intersection and multiple points."
-msgstr "Контур не может быть исправлен от проблемы самопересечения и дублирующихся точек."
+msgstr "Не удалось исправить дублирующиеся точки и самопересечение контура."
```
```suggestion
msgstr "Не удалось исправить контур из-за самопересечения и дублирующихся точек."
```

AndylgTom commented on localization/i18n/ru/OrcaSlicer_ru.po:
```diff
@@ -1572,13 +1571,13 @@ msgstr "Загрузка настроек"
 #, c-format, boost-format
 msgid "Click to download new version in default browser: %s"
-msgstr "Нажмите OK, чтобы загрузить последнюю версию в браузере по умолчанию: %s"
+msgstr "Нажмите «Скачать» для загрузки версии %s в вашем основном браузере."
```
```suggestion
msgstr "Нажмите «Скачать» для загрузки новой версии %s в вашем браузере по умолчанию."
```

AndylgTom replied:
Стандартный виндовский термин, незачем велосипед изобретать

Felix14-v2 replied:
Там места не хватает (на моём мониторе, по крайней мере), поэтому пытался сжать

AndylgTom commented on localization/i18n/ru/OrcaSlicer_ru.po:
```diff
@@ -1572,13 +1571,13 @@ msgstr "Загрузка настроек"
 
 msgid "This is the newest version."
-msgstr "У вас стоит последняя версия программы."
+msgstr "Установлена новейшая версия программы."
```
```suggestion
msgstr "У вас стоит последняя версия программы."
```

AndylgTom replied:
никто так не говорит «новейшая версия».

Felix14-v2 replied:
Ну вообще, регулярно встречаю такое употребление. Тут не принципиально, только `У вас стоит ...` уберу :D

AndylgTom commented on localization/i18n/ru/OrcaSlicer_ru.po:
```diff
@@ -1588,9 +1587,9 @@ msgid ""
-"Возможно, файл конфигурации OrcaSlicer повреждён и не может быть обработан.\n"
-"OrcaSlicer попытался воссоздать файл конфигурации.\n"
-"Обратите внимание, что настройки приложения будут потеряны, но профили принтера не будут затронуты."
+"Не удалось прочитать файл настроек OrcaSlicer из-за возможного повреждения.\n"
```
Что-то все равно какая-то корявая формулировка «попытался пересоздать»... Он когда это пишет, то это значит УЖЕ его пересоздал. Если что-то не то с OrcaSlicer.conf, программа пересоздаёт/восстанавливает его, основываясь на данных из резервной копии *.bak, если резервной копии нету, то генерирует новый, в котором как раз всё по умолчанию — это и есть этот случай.

Наверное, если развернуто, имеется в виду типа... OrcaSlicer попытался восстановить файл настроек, но сделать это не получилось, поэтому файл настроек был создан так сказать с дефолтными параметрами.

AndylgTom commented on localization/i18n/ru/OrcaSlicer_ru.po:
```diff
@@ -1705,7 +1704,7 @@ msgid "Infill"
 
 msgid "Support"
-msgstr "Поддержка"
+msgstr "Поддержки"
```
Так пишу потому что подразумевая что Поддержка - это как общая структура (также как и Подложка), а не отдельные мелкие поддержки.

Felix14-v2 replied:
Структурно – не соглашусь, т.к. `Поддержка` воспринимается именно как что-то в единственном экземпляре, а тут мы настраиваем все существующие (или возможно существующие) поддержки, которых у одной детали может быть несколько (в случае с древовидной поддержкой их вообще почти всегда несколько). Да и деталей может быть больше одной, опять же. Поэтому так просто уместнее.

AndylgTom commented on localization/i18n/ru/OrcaSlicer_ru.po:
```diff
@@ -2139,22 +2138,22 @@ msgid "Right button click the icon to drop the object settings"
 
 msgid "Click the icon to toggle printable property of the object"
-msgstr "Нажмите на значок, чтобы разрешить/запретить печать модели."
+msgstr "Переключить печать модели"
```
Что значит переключить печать? По дурацки же звучит, тоже самое что сказать «Выключить зонтик»

Felix14-v2 replied:
Переключить – изменить состояние (с `вкл` на `выкл` и наоборот), буквально по определению...

Но если так посмотреть, тут по контексту это слово можно вообще опустить, у нас же кнопка и так является переключателем:
<img width="845" height="265" alt="{0130695D-3CEC-400B-BC70-D597EDB181CB}" src="https://github.com/user-attachments/assets/b04f4bc4-b801-4c98-91a1-a721b5cb43fc" />

Если печать отключена, то эта надпись воспринимается как действие (Печатать → "включить печать"), а если уже включена – как сообщение о состоянии.

AndylgTom commented on localization/i18n/ru/OrcaSlicer_ru.po:
```diff
@@ -2139,22 +2138,22 @@ msgid "Right button click the icon to drop the object settings"
 
 msgid "Click the icon to toggle printable property of the object"
-msgstr "Нажмите на значок, чтобы разрешить/запретить печать модели."
+msgstr "Переключить печать модели"
```
```suggestion
msgstr "Разрешить/запретить печать модели"
```

AndylgTom commented on localization/i18n/ru/OrcaSlicer_ru.po:
```diff
@@ -2139,22 +2138,22 @@ msgid "Right button click the icon to drop the object settings"
 
 msgid "Click the icon to edit support painting of the object"
-msgstr "Нажмите на значок, чтобы изменить области расположения поддержек модели."
+msgstr "Разместить поддержки"
```
Так значек появляется только когда уже были добавлены поддержки, и их можно как разместить так и стереть.

AndylgTom replied:
```suggestion
msgstr "Изменить области расположения поддержек"
```

AndylgTom commented on localization/i18n/ru/OrcaSlicer_ru.po:
```diff
@@ -2139,22 +2138,22 @@ msgid "Right button click the icon to drop the object settings"
 
 msgid "Click the icon to edit color painting of the object"
-msgstr "Нажмите на значок, чтобы изменить цветовую раскраску модели."
+msgstr "Перекраскрасить модель"
```
```suggestion
msgstr "Изменить цветовую раскраску модели"
```

AndylgTom commented on localization/i18n/ru/OrcaSlicer_ru.po:
```diff
@@ -2810,13 +2809,13 @@ msgstr "Orca Slicer основан на PrusaSlicer и BambuStudio"
-
 msgid "This software uses open source components whose copyright and other proprietary rights belong to their respective owners"
-msgstr "В данном программном обеспечении используются компоненты с открытым исходным кодом, авторские и иные права на которые принадлежат их соответствующим владельцам."
+msgstr "В данном программном обеспечении используются компоненты с открытым исходным кодом, авторские и иные права на которые принадлежат их соответствующим владельцам"
```
```suggestion
msgstr "В данном программном обеспечении используются компоненты с открытым исходным кодом, авторские и иные права на которые принадлежат следующим соответствующим владельцам"
```

AndylgTom commented on localization/i18n/ru/OrcaSlicer_ru.po:
```diff
@@ -2810,13 +2809,13 @@ msgstr "Orca Slicer основан на PrusaSlicer и BambuStudio"
 #, c-format, boost-format
 msgid "About %s"
-msgstr "О %s"
+msgstr "Об %s"
```
О программе %s
```suggestion
msgstr "О программе %s"
```

AndylgTom commented on localization/i18n/ru/OrcaSlicer_ru.po:
```diff
@@ -2977,7 +2976,7 @@ msgid "Enable AMS"
 
 msgid "Print with filaments in the AMS"
-msgstr "Печать пластиковыми нитями из АСПП."
+msgstr "Печать катушками из АСПП."
```
```suggestion
msgstr "Печать пластиковыми нитями с катушек из АСПП."
```

AndylgTom commented on localization/i18n/ru/OrcaSlicer_ru.po:
```diff
@@ -4127,11 +4126,12 @@ msgstr "Переключиться на обычный режим"
 
 msgid "Quality / Speed"
-msgstr "Качество / Скорость"
+msgstr "Качество / cкорость"
```
```suggestion
msgstr "Качество/Скорость"
```

AndylgTom replied:
Слитно исправил + с большой буквы, а не с маленькой

AndylgTom commented on localization/i18n/ru/OrcaSlicer_ru.po:
```diff
@@ -4167,25 +4168,25 @@ msgid "Smoothing"
 
 msgid "object selection"
-msgstr "Выбор модели"
+msgstr "Выделить модель"
```
```suggestion
msgstr "Выбрать модель"
```

AndylgTom commented on localization/i18n/ru/OrcaSlicer_ru.po:
```diff
@@ -4167,25 +4168,25 @@ msgid "Smoothing"
 
 msgid "part selectiont"
-msgstr "Выбор элемента"
+msgstr "Выделить часть"
```
```suggestion
msgstr "Выбрать элемент"
```

AndylgTom replied:
Мож элемент все же уместнее? 
Обязательно, что ли, чтобы в сборке каждый объект был частью одной модели? Сборка может быть из нескольких элементов, которые могут не являться частями друг другу, никак не пересекаются и т. д.
Т. е. они хоть и являются частью общей сборки, но они не являются частью друг друга (как, например, разделенная на части модель).

AndylgTom commented on localization/i18n/ru/OrcaSlicer_ru.po:
```diff
@@ -4167,25 +4168,25 @@ msgid "Smoothing"
 
 msgid "number keys"
-msgstr "цифровые клавиши"
+msgstr "цифры"
```
```suggestion
msgstr "(цифры)"
```

Felix14-v2 committed:
Update OrcaSlicer_ru.po

AndylgTom commented on localization/i18n/ru/OrcaSlicer_ru.po:
```diff
@@ -5739,28 +5741,28 @@ msgid "Material settings"
 
 msgid "Auto orient objects on current plate"
-msgstr "Автоориентация моделей на текущей печатной пластине"
+msgstr "Положить модели на стол"
```
В таком смыле двоякий смысл, как будто они висят где то в воздухе,  а смысл то в другом

Felix14-v2 replied:
> как будто они висят где то в воздухе

Не обязательно – модели могут стоять на столе вертикально или неустойчиво, и эта кнопка именно что кладёт их наиболее устойчивым образом. К тому же, если не ошибаюсь, в Cura эта кнопка именно так и называется (`Положить на стол`).
Значок у кнопки как раз передаёт смысл придачи устойчивого положения:
<img width="166" height="172" alt="{9E5881DB-66AF-467B-B176-4F779A26A5C5}" src="https://github.com/user-attachments/assets/af7cdb39-8afe-4432-8ea2-4e6633f6c299" />

Другой момент, что функция работает очень криво, но это уже косяк не в локализации.

AndylgTom commented on localization/i18n/ru/OrcaSlicer_ru.po:
```diff
@@ -6768,10 +6770,10 @@ msgid "AMS filaments"
 
 msgid "Click to select filament color"
-msgstr "Нажмите, чтобы выбрать цвет прутка"
+msgstr "Изменить цвет катушки"
```
Да не может быть у катушки цвета, цвет у прутка на катушке

Felix14-v2 replied:
Согласен, я бы вообще тут оставил просто `Изменить цвет`, и так понятно, что кнопка делает
<img width="306" height="103" alt="{0EC1234C-6BD2-4740-A6F5-9D7571BABC7E}" src="https://github.com/user-attachments/assets/3821a5fc-7369-495c-9767-abd3c0912639" />

Felix14-v2 committed:
Update OrcaSlicer_ru.po

Felix14-v2 committed:
Update OrcaSlicer_ru.po

Co-authored-by: Andy <andylg@yandex.ru>

Felix14-v2 committed:
Update OrcaSlicer_ru.po

Felix14-v2 committed:
Merge branch 'main' of https://github.com/Felix14-v2/Andy-OrcaSlicer

Felix14-v2 committed:
Update OrcaSlicer_ru.po

Co-authored-by: Andy <andylg@yandex.ru>

Felix14-v2 committed:
Update OrcaSlicer_ru.po

Felix14-v2 committed:
Update OrcaSlicer_ru.po

Felix14-v2 committed:
Update OrcaSlicer_ru.po

Felix14-v2 committed:
Merge branch 'main' of https://github.com/Felix14-v2/Andy-OrcaSlicer

Felix14-v2 committed:
Update OrcaSlicer_ru.po

Felix14-v2 committed:
Update OrcaSlicer_ru.po

Felix14-v2 committed:
Update OrcaSlicer_ru.po

Felix14-v2 committed:
Merge branch 'main' of https://github.com/Felix14-v2/Andy-OrcaSlicer

Felix14-v2 committed:
Update OrcaSlicer_ru.po

Felix14-v2 committed:
Update OrcaSlicer_ru.po

Felix14-v2 committed:
Update OrcaSlicer_ru.po

Felix14-v2 committed:
Merge branch 'main' of https://github.com/Felix14-v2/Andy-OrcaSlicer

Felix14-v2 reviewed (commented):
Гитхабу плохо, отпишусь так

Felix14-v2 committed:
Update OrcaSlicer_ru.po

Felix14-v2 reviewed (commented):
Ещё комментарии

Felix14-v2 committed:
Update OrcaSlicer_ru.po

Felix14-v2 committed:
Attempt to fix sidebar tabs

by shrinking their width

Felix14-v2 committed:
Update OrcaSlicer_ru.po

Felix14-v2 committed:
Consistency nuke

- Cleanup based on glossary
- A lot of fixes (and workarounds)
All to make translation consistent and easy to read/understand

Felix14-v2 committed:
Финальные правки
